pom.xml

```diff
@@ -420,19 +427,18 @@
                 <containerNamePattern>hauth</containerNamePattern>
                 <ports>
                   <!-- Start test container on an available host port -->
-                  <port>${test.http.port}:8888</port>
+                  <port>${test.http.port}:${test.http.port}</port>
```

Comment:
What is the value to overriding the containers' internal ports? We set them on the outside. Setting them to the same thing on the inside seems like unnecessary configuration.

Reply:
None really I guess, I can revert